lib/scrum/priority_name.rb

```diff
@@ -0,0 +1,15 @@
+module Scrum
+  class PriorityName
```

Comment:
Yes, that's a good place to class methods. Honestly I just moved these methods here quickly, because they were part of the enormous `Card` class. 
I thought this might become a real object later, but that's currently not necessary at all.